cmd/tempo-query/Dockerfile

```diff
@@ -1,9 +1,10 @@
 FROM jaegertracing/jaeger-query:1.43.0
 
 ENV SPAN_STORAGE_TYPE=grpc-plugin \
-    GRPC_STORAGE_PLUGIN_BINARY=/tmp/tempo-query
+    GRPC_STORAGE_PLUGIN_BINARY=/tempo-query
+
+# Ensure /tmp dir exists, hashicorp plugins need a /tmp dir to exist.
+RUN mkdir -p /tmp
 
-# This is silly, but it's important that tempo-query gets copied into /tmp
```

Comment:
you didn't like my cheeky comment ;) ?

Reply:
oh, @electron0zero suggested that!

Reply:
Haha I personally liked it and that’s why I initially kept that part in there 😄

Reply:
I was okay with either but wanted to sound like we know what we are doing 😆